postgres/src/main/java/org/jdbi/v3/postgres/PGObjectArrayArgumentFactory.java

```diff
@@ -0,0 +1,49 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.jdbi.v3.postgres;
+
+import java.sql.Array;
+import java.sql.Types;
+
+import org.jdbi.v3.core.argument.AbstractArgumentFactory;
+import org.jdbi.v3.core.argument.Argument;
+import org.jdbi.v3.core.argument.internal.strategies.LoggableBinderArgument;
+import org.jdbi.v3.core.config.ConfigRegistry;
+import org.postgresql.PGConnection;
+import org.postgresql.util.PGobject;
+
+/**
+ * Argument factory for array of {@link PGobject}.
+ */
+public class PGObjectArrayArgumentFactory extends AbstractArgumentFactory<PGobject[]> {
```

Comment:
Jdbi already provides array support--you just have to register an array type and tell Jdbi the SQL type name that goes with the Java class:

```java
jdbi.registerArrayType("point", PGpoint.class);
```

Reply:
@qualidafial at which point should we do this(`jdbi.registerArrayType("point", PGpoint.class)`)?